packages/desktop_webview_window/windows/web_view.cc

```diff
@@ -144,14 +146,34 @@ void WebView::OnWebviewControllerCreated() {
                 std::make_unique<flutter::EncodableValue>(flutter::EncodableMap{
                     {flutter::EncodableValue("id"), flutter::EncodableValue(web_view_id_)},
                 }));
-            LPWSTR uri;
-            args->get_Uri(&uri);
-            method_channel_->InvokeMethod(
-                "onUrlRequested",
-                std::make_unique<flutter::EncodableValue>(flutter::EncodableMap{
-                    {flutter::EncodableValue("id"), flutter::EncodableValue(web_view_id_)},
-                    {flutter::EncodableValue("url"), flutter::EncodableValue(wide_to_utf8(std::wstring(uri)))},
-                }));
+
+            if (triggerOnUrlRequestedEvent) {
```

Comment:
Is this necessary? It should make more sense to trigger onUrlRequest on any request (including from navigate(String) method).

Reply:
In my opinion its more convenient like this, because otherwise the call of "webview.launch(url)" would trigger the OnUrlRequest event every time and you manually had to grant it every time. If you call "webview.launch(url)" you know already that the webview will navigate to the url now, because you requested it yourself. But I am more interested in navigations triggered by the webview itself.

Reply:
sorry for later. 

But I still stand by my opinion.

for example: 
If you need to display the currently requested URL in the console or elsewhere, you only need to add a callback instead of adding code in various places.

Reply:
Ok seems also reasonable. Then I suggest to add an optional parameter with default value to webview.launch:
 webview.launch(..., triggerOnUrlRequestEvent=true)

Reply:
I added a new commit containing the additional parameter for webview.launch

Please review